app/controllers/nominations/nominate_induction_coordinator_controller.rb

```diff
@@ -60,6 +66,12 @@ def nominate_induction_tutor_form_params
     params.require(:nominate_induction_tutor_form).permit(:full_name, :email, :token)
   end
 
+  def record_nomination_email_opened
+    NominationEmail
+      .where(token: @nominate_induction_tutor_form.token, opened_at: nil)
```

Comment:
Reading this gives the impression that you are updating multiple records which is not the case. Why not just use find_by with update?

Reply:
Yeah, I was on the fence here. Since I only do the update here, I do not need to instantiate AR object at all and can just o a single `UPDATE` query (instead of two: SELECT and UPDATE). Additionally, as written it will automatically bypass all the model validations, which is, surprisingly, correct in this context (I don't want to miss this write only because we added new validation on the model?) - this will probably never be a case though.

Reply:
So the part about number of queries is interesting. I would probably err on the side of code readability and expressiveness over database optimisations that save 1 query. Because premature optimisation is evil - I imagine we will have load tests, and I am 99% sure that this is not gonna be the place that cracks first.
Saving N+1 queries, well, that would be more compelling. 

As for bypassing validation, it looks like an evil side effect of inner Rails machinations. We should have tests for this flow (both in cucumber and request tests) one of which should test whether this email was updated.

Reply:
I completely agree with @kicferk1 In addition, the moment someone adds validations to this model - which is very likely - that will be the first thing to break.